frontend/src/cards/TableCard.tsx

```diff
@@ -73,6 +75,29 @@ export function TableCard(props: TableCardProps) {
                 />
               </CardContent>
             )}
+            {!queryResponse.dataIsMissing() && props.breakdownVar === RACE && (
```

Comment:
Checkin rq: will this only show for COVID data?

Reply:
oh good catch! adding check for covid